test/e2e/common/apparmor.go

```diff
@@ -67,6 +67,18 @@ elif ! grep "%[3]s" /proc/self/attr/current; then
   exit 3
 fi`, appArmorDeniedPath, appArmorAllowedPath, appArmorProfilePrefix+f.Namespace.Name)
 
+	if unconfined {
+		profile = apparmor.ProfileNameUnconfined
+		testCmd = `
+if cat /proc/sysrq-trigger | grep "Permission denied"; then
```

Comment:
This won't work because error is printed to stderr and this command is grepping on stdout. At least it should be `cat /proc/sysrq-trigger 2>&1 | grep "Permission denied"`.

Also there is a simpler way to check whether a user has access to read a file:
```bash
if [ ! -r /proc/sysrq-trigger ]; then
    echo "FAILURE: reading /proc/sysrq-trigger should be allowed"
fi
```

I'd also suggest to
- print errors to stderr: `echo >&2 "FAILURE..."`
- prefer using single quotes over double quotes when you don't have variables inside a string: `echo 'FAILURE'`

Reply:
> Also there is a simpler way to check whether a user has access to read a file:

@php-coder We just want the error message, not really read a file.